specification/ai/Azure.AI.Projects/datasets/routes.tsp

```diff
@@ -30,6 +31,7 @@ interface Datasets
     {
       @doc("The pending upload request parameters")
       @Http.bodyRoot
+      @clientName("pendingUploadOptions")
```

Comment:
Is this necessary alongside the change in client.tsp? If it is, do we have it for all the others? If it's not, can we choose which place and remain consistent?